MAUI/TextInputLayout/Custom-Icons.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Custom Icons in .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here all about Custom Icons support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Custom Icons in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+Any custom icons can be added to the leading edge or the trailing edge of input view in the text input layout control. The events and commands related to the custom icons should be handled at the application level.
```

Comment:
Any custom icons can be added to the leading edge or the trailing edge of the input view in the text input layout control. The events and commands related to the custom icons should be handled at the application level.

Reply:
Addressed the changes